lib/src/pubsub_impl.dart

```diff
@@ -5,15 +5,13 @@
 part of gcloud.pubsub;
 
 class _PubSubImpl implements PubSub {
-  final http.Client _client;
   final String project;
   final pubsub.PubsubApi _api;
   final String _topicPrefix;
   final String _subscriptionPrefix;
 
-  _PubSubImpl(client, project)
-      : this._client = client,
-        this.project = project,
+  _PubSubImpl(http.Client client, String project)
```

Comment:
Can't we use `this.project` now and still use it in the initializers?

Reply:
ack

Reply:
ack